### ISSUE TITLE
Sirocco Kukri interacts incorrectly with Puk's wind damage>healing conversion

### ISSUE THREAD

**Issue by kassisaf** [EdenServer/community#1340]
Puks passively convert incoming wind damage to cures, which appears to be working correctly with spells.

With Sirocco Kukri's added effect however, it prints a large amount of healing to the log (130k+) but doesn't appear to actually heal. Not sure if this is just a bad message or a bad interaction as well, as it may be healing a small amount that is unnoticeable after the initial attack damage.

![Jody_2019 03 19_161150](https://user-images.githubusercontent.com/9102794/54648124-436d5580-4a62-11e9-9cb9-07e51e3fa894.png)

**Comment by Serephim** [comment]
Looks like the log bug that happened with holy bolts.

**Comment by E5una** [comment]
The same is true for Detonation. I've done a few self skillchains on Puks to create Detonation and it always prints a numbers close to 65k. The number printed isn't actual healing or damaging, so it just seems like a logging issue.

![image](https://user-images.githubusercontent.com/13668758/73294103-26052e00-41d3-11ea-8dc7-dd84bd237819.png)


FYI I've used Mysterious Light to deal wind damage to Puks and they did not heal at all. I saw it's HP move down 4%.

![image](https://user-images.githubusercontent.com/13668758/73294167-49c87400-41d3-11ea-8e47-4b3adc169e22.png)
